### ISSUE TITLE
can't get the same accuracy in the paper

### ISSUE THREAD

**Issue by venusafroid** [songyouwei/ABSA-PyTorch#38]
Hi,
Thank you for your modification of the error. I try to run the aen model, and I get 0.7642... after 50 epoch, I am wondering why.
I will be grateful if I can get your reply.

**Comment by songyouwei** [comment]
Which dataset and model do you use?

**Comment by venusafroid** [comment]
aen model and restaurant dataset

**Comment by songyouwei** [comment]
Small dataset is sensitive to hyperparameter settings and training procedure is not very stable. Hyperparams should be carefully determined, especially learning rate, dropout and num_epochs.
